src/commands/login.ts

```diff
@@ -0,0 +1,27 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../oclif/CustomCommand'
+import authLogin from '../modules/auth/login'
+
+export default class Login extends CustomCommand {
+  static description = 'Log into a VTEX account'
+
+  static examples = ['vtex login', 'vtex login storecomponents', 'vtex login storecomponents myworkspace']
+
+  static flags = {
+    help: oclifFlags.help({ char: 'h' }),
+  }
+
+  static args = [
+    { name: 'account', required: false },
+    { name: 'workspace', required: false },
```

Comment:
Maybe workspace should be a flag? I think in the current version the workspace has to be passed as a flag

Reply:
I wanted to make a standard, since currently the distinctions between what should be a flag and what should be an argument is obscure for me. But we can improve that in the new major, for now it is better to keep the previous behaviour.